SimplenoteUITests/SimplenoteUITestsLogin.swift

```diff
@@ -0,0 +1,82 @@
+import XCTest
+
+class SimplenoteUISmokeTestsLogin: XCTestCase {
+	let testDataInvalidEmail = "user@gmail."
+	let testDataNotExistingEmail = "nevergonnagiveyouup@gmail.com"
+	let testDataInvalidPassword = "ABC"
+	let testDataNotExistingPassword = "ABCD"
+
+	override class func setUp() {
+		app.launch()
+	}
+
+	override func setUpWithError() throws {
+		continueAfterFailure = true
```

Comment:
I was going to suggest to move this into the `class` `setUp()` because we only need to set it once (unless we want to change it to `false` at some point during the test run).

I looked [at the docs](https://developer.apple.com/documentation/xctest/xctestcase/1496260-continueafterfailure) to make sure my assumption was true and discovered that `true` is the default value. Really, I should have known that because that's what my tests do all the time 🤣 .

Given it's the default value, what do you think about removing this explicit setting?

If you'd like to keep it because it might be handy to toggle it while debugging a test failure, then I'd suggest to add it as a commented code to the `class` setup.

```swift
// If you need to debug a failing test and don't want to waste time with the test suite 
// continuing after it fails, uncomment the following line.
//continueAfterFailure = false
```

Reply:
I was mistakenly thinking that having this set to `false` will actually cancel all following tests is a class. I really had no reason to have a failing test itself to continue the execution. Removed it from everywhere. Thanks!